src/config.rs

```diff
@@ -166,6 +167,29 @@ impl IndexMut<usize> for ColorList {
     }
 }
 
+#[derive(Debug, Deserialize)]
+pub struct Shell<'a> {
+    program: Cow<'a, str>,
+    args: Vec<String>,
```

Comment:
The compiler doesn't complain because the struct is still valid. The derived `Deserialize` impl turns a missing `args` into a deserialization error. Adding the attribute will change the missing field action to use the `Default` for that field.